proto/process/agent.proto

```diff
@@ -857,6 +857,31 @@ message DNSEntry {
 	repeated string names = 1;
 }
 
+enum K8sResources {
+    unsetNodeType = 0;
+    pod = 1;
+    replicaSet = 2;
+    service = 3;
+    node = 4;
+    cluster = 5;
+    job = 6;
+    cronJob = 7;
+    daemonSet = 8;
+    statefulSet = 9;
+    persistentVolume = 10;
+    persistentVolumeClaim = 11;
+    role = 12;
+    roleBinding = 13;
+    clusterRole = 14;
+    clusterRoleBinding = 15;
+    serviceAccount = 16;
+    ingress = 17;
+    deployment = 18;
+    namespace = 19;
+    crd = 20;
+    cr = 21;
```

Comment:
we should use upper case format for values here as specified in https://developers.google.com/protocol-buffers/docs/style

Reply:
Thanks for the feedback, I addressed in c440357cc194137632900f8dfca8aceee35a6575

I don't add prefix in proto, such as `K8SRESOURCE_POD`, because in generated file we can have `K8sResource_K8SRESOURCE_POD` making me feel the name is too long